db/re/skill_db.yml

```diff
@@ -19234,16 +19234,16 @@ Body:
     AfterCastActDelay: 2000
     Duration1:
       - Level: 1
-        Time: 7000
+        Time: 10000
       - Level: 2
-        Time: 9000
+        Time: 12000
       - Level: 3
-        Time: 11000
+        Time: 14000
       - Level: 4
-        Time: 13000
+        Time: 16000
```

Comment:
This reverts our fixes from last commit, don't forget that 3s fixed duration is added!

Reply:
You're right! This was a mixup on my part merging with master!